server/sonar-webserver-webapi/src/main/java/org/sonar/server/issue/ws/SearchAction.java

```diff
@@ -365,6 +366,9 @@ public void define(WebService.NewController controller) {
       .setRequired(false)
       .setExampleValue("'Europe/Paris', 'Z' or '+02:00'")
       .setSince("8.6");
+    action.createParam(PARAM_ORGANIZATION)
+      .setDescription("Organization key")
+      .setInternal(true);
```

Comment:
Please explain why the setInternal(true) is needed.

Reply:
It's for documentation. This parameter will be marked as INTERNAL and will be visible only with `Show internal API` option enabled.

![image](https://github.com/codescan-io/sonarqube/assets/556093/ad5ab15b-e407-4058-902d-b12be1c33460)